layouts/partials/head.html

```diff
@@ -13,10 +13,17 @@
   {{- template "_internal/google_news.html" . -}}
   {{- template "_internal/schema.html" . -}}
   {{- template "_internal/twitter_cards.html" . -}}
-  {{ if eq (getenv "HUGO_ENV") "production" }}
-  {{ template "_internal/google_analytics.html" . }}
-  {{ end }}
   {{ partialCached "head-css.html" . "asdf" }}
+{{ if eq (getenv "HUGO_ENV") "production" }}
+<!--This function is from tagmanager.google.com's installation instructions -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ .Site.Params.google_tag_id }}"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', "{{ .Site.Params.google_tag_id }}");
+</script>
+{{end}}
```

Comment:
Cameron - any custom code that needs to be in the header should go into layouts/partials/hooks/head-end.html.